### ISSUE TITLE
Horizontal scroll bar in the English version of the text

### ISSUE THREAD

**Issue by akatsoulas** [mozilla/sumo-project#362]
Screenshot
https://support.mozilla.org/de/gallery/image/37432

**Comment by kkellydesign** [comment]
Can you provide a URL for this?

**Comment by philipp-sumo** [comment]
the screenshot is from https://support.mozilla.org/de/kb/suchvorschlaege-in-firefox/edit after clicking the "Toggle Diff" link

**Comment by kkellydesign** [comment]
Thanks @philipp-sumo. I found the issue here, and there's not a good way to fix this. The issue is on line 19 of that text, where there are no spaces in that whole string of text, so the browser sees that as a "word" and makes the entire table stretch to fit that cell. 

<img width="484" alt="Screen Shot 2020-04-16 at 10 29 31 AM" src="https://user-images.githubusercontent.com/202590/79468686-3a381a80-7fcd-11ea-9fe3-b9405f15084d.png">

I did make a minor change so the fonts are the same between these two different text formatting plugins, so that will reduce the amount of scrolling somewhat on this particular page. But it doesn't really fix the problem. 

<img width="950" alt="Screen Shot 2020-04-16 at 10 34 17 AM" src="https://user-images.githubusercontent.com/202590/79469292-fd205800-7fcd-11ea-876a-ef53eaec2736.png">
